tests/Db/Command/CommandTest.php

```diff
@@ -39,17 +40,20 @@ public function testAddCheck(): void
         );
     }
 
-    public function testAddColumn(): void
+    /** @dataProvider \Yiisoft\Db\Tests\Provider\CommandProvider::columnTypes */
+    public function testAddColumn(ColumnInterface|string $type): void
     {
         $db = $this->getConnection();
 
         $command = $db->createCommand();
-        $sql = $command->addColumn('table', 'column', SchemaInterface::TYPE_INTEGER)->getSql();
+        $sql = $command->addColumn('table', 'column', $type)->getSql();
+
+        $type = $db->getQueryBuilder()->getColumnType($type);
```

Comment:
Suggest use another name for variable. $type here and $type in method parameters are confused =)